AcademyResidentInformationApi/V1/UseCase/GetAllClaimantsUseCase.cs

```diff
@@ -14,9 +15,12 @@ public GetAllClaimantsUseCase(IAcademyGateway academyGateway)
             _academyGateway = academyGateway;
         }
 
-        public ClaimantInformationList Execute()
+        public ClaimantInformationList Execute(ClaimantQueryParam cqp, int cursor, int limit)
         {
-            var claimants = _academyGateway.GetAllClaimants();
+            limit = limit < 10 ? 10 : limit;
+            limit = limit > 100 ? 100 : limit;
```

Comment:
I might be missing them but I don't think are any tests around the limit functionality

Reply:
I've added tests relating to the limit